feature-flag/src/main/kotlin/com/linecorp/android/featureflag/FeatureFlagJavaFileWriter.kt

```diff
@@ -325,5 +325,7 @@ internal class FeatureFlagJavaFileWriter(
 
     companion object {
         private const val FEATURE_FLAG_CLASS_NAME = "FeatureFlag"
+
+        private const val DEPRECATED_ANNOTATION_CLASS_NAME = "Deprecated"
```

Comment:
Please add the following comment
```kotlin
// We specify annotation by name instead of `Class` to use `java.lang` package rather than `kotlin`.
```